### DISCUSSION TITLE
Add test command?

### DISCUSSION THREAD

**Discussion by Siecje** [astral-sh/rye#649] [Ideas]
It would be nice to know the test invocation for any rye project.

**Comment by mitsuhiko** [comment]
Yes, definitely something I want to add. The issue is to some degree finding out what the right test runner is / should be.

**Reply by Siecje** [reply]
As a default? I assume it would be overridden by a config.

```
[tool.rye]
test = "pytest tests/"
```

I don't care which test runner the project uses, I just want to be able to `rye test` to run the tests.

I'm also curious of your opinion of pytest.
